structure-mappings.json

```diff
@@ -151,8 +151,8 @@
   "radio-button-unchecked": "circle",
   "random": "random",
   "redirect": "redirect",
-  "refresh-alert": "refresh-alert",
-  "refresh-default": "reload",
+  "refresh-alert": "sync-alert",
+  "refresh-default": "sync",
```

Comment:
"reload" is actually the correct mapping here. 
Structure Refresh - Default: https://github.com/hashicorp/structure-icons/blob/master/src/Refresh%20-%20Default.svg
Flight Reload: https://flight-hashicorp.vercel.app/?query=reload

Reply:
Ah thanks Heather! Fixed that up

---

I spotted a few more that where missing also (just 3 brand icons), shall I just pop one more commit on the end of this PR, then you can check that commit also before merge (might be easier than making a fresh PR just for that?)

Reply:
Doh 🤦 thanks for the suggestion reload/refresh 😆  more or less the same 😬 😄